content/02.main-text.md

```diff
@@ -269,11 +268,12 @@ For example, open access publishing provides an economic model that encourages m
 More broadly, open licensing solves legal barriers to content reuse, enabling any type of scholarly output to become part of the commons [@doi:10.1109/MCSE.2009.19; @doi:10.1038/536016a].
 The opportunity to reuse data and code for new investigations, as well as a push for increased reproducibility, has begot a movement to make all research outputs public, unless there are bonafide privacy or security concerns [@doi:10.1126/science.aah6168; @doi:10.1038/nature10836; @doi:10.1371/journal.pbio.1001195].
 New tools and services make it increasingly feasible to publicly share the unabridged methods of a study, especially for computational research, which consists solely of software and data.
-Greater openness in both research methods and publishing is creating an opportunity to redefine peer review and the role journals play in communicating science [@doi:10.12688/f1000research.12037.1].
+
+Greater openness in both research methods and publishing creates an opportunity to redefine peer review and the role journals play in communicating science [@doi:10.12688/f1000research.12037.2].
 At the extreme is real-time open science, whereby studies are performed entirely in the open from their inception [@doi:10.1242/dmm.003285].
 Many such research projects have now been completed, benefiting from the associated early-stage peer review, additional opportunity for online collaboration, and increased visibility [@doi:10.3897/rio.1.e7547; @doi:10.7554/eLife.26726].
 
-Manubot is designed to be an authoring protocol for real-time open science, with especial relevance for projects that are already using the open source software workflow to manage their research.
+Manubot is an ideal authoring protocol for real-time open science, especially for projects that are already using an open source software workflow to manage their research.
 While Manubot does require technical expertise, the benefits are manyfold.
 Specifically, Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
-We hope that these attributes help integrate Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
+These attributes make it possible to integrate Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
```

Comment:
Doulble "make" in this sentence. Not that I think that is a big problem. Just pointing it out. @agitter feel free to leave if you think it's OKAY.